static/gsApp/components/gsBanner.tsx

```diff
@@ -825,179 +811,6 @@ class GSBanner extends Component<Props, State> {
     });
   }
 
-  renderOverageAlert(isWarning: boolean) {
-    const {organization, subscription} = this.props;
-    const plan = subscription.planDetails;
-    let overquotaPrompt: React.ReactNode;
-    let eventTypes: EventType[] = [];
-
-    if (prefersStackedNav(organization)) {
-      // new nav uses sidebar quota alert (see quotaExceededNavItem.tsx)
-      return null;
```

Comment:
This was not being rendered in the new nav so can be removed (which I did to fix tests)